flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/filesystem/FileSystemTableSource.java

```diff
@@ -121,6 +127,14 @@ public TableSchema getSchema() {
 				return schema;
 			}
 
+			@Override
+			public TypeInformation<?> createTypeInformation(DataType dataType) {
```

Comment:
We don't need type info now.

Reply:
JsonRowDataDeserializationSchema need this type info

Reply:
we can pass a GenericTypeInformation